website/src/docs/hotchocolate/Security/authorization.md

```diff
@@ -0,0 +1,276 @@
+---
+title: Authorization
+---
+
+# Authentication
+
+GraphQL as defined by the spec does not specify how a user has to authenticate against a schema in order to execute queries. GraphQL does not even specify how requests are sent to the server using HTTP or any other protocol. _Facebook_ specified GraphQL as transport agnostic, meaning GraphQL focuses on one specific problem domain and does not try to solve other problems like how the transport might work, how authentication might work or how a schema implements authorization. These subjects are considered out of scope.
+
+If we are accessing GraphQL servers through HTTP then authenticating against a GraphQL server can be done in various ways and Hot Chocolate does not prescribe any particular.
+
+We basically can do it in any way ASP.NET core allows us to.
+
+[Overview of ASP.NET Core authentication](https://docs.microsoft.com/en-us/aspnet/core/security/authentication/?view=aspnetcore-3.1)
+
+# Authorization
+
+Authorization on the other hand is something Hot Chocolate can provide some value to by introducing a way to authorize access to fields with the `@authorize`-directive.
+
+But let's start at the beginning with this. In order to add authorization capabilities to our schema add the following package to our project:
+
+```bash
+dotnet add package HotChocolate.AspNetCore.Authorization
+```
+
+In order to use the `@authorize`-directive we have to register it like the following with our schema:
+
+```csharp
+SchemaBuilder.New()
+  ...
+  .AddAuthorizeDirectiveType()
+  ...
+  .Create();
+```
+
+Once we have done that we can add the `@authorize`-directive to object types or their fields.
+
+The `@authorize`-directive on a field takes precedence over one that is added on the object type definition.
+
+SDL-First:
```

Comment:
I think the ExampleTabs would be better suited for the code examples in these documents.